### PR TITLE
FEATURE: Replace combo-box with search filter

### PR DESCRIPTION
**Pull request by rishabhnambiar**
---

### CODE REVIEW

File: assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6 [hunk 1]
```diff
@@ -26,6 +26,16 @@ export default Ember.Controller.extend({
   sortBy: ["last_run_at:desc"],
   sortedQueries: Em.computed.sort("model", "sortBy"),
 
+  searchedContent: function() {
```
Comment (techAPJ) [inline assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6]: We should use `computed` along with ES6 function syntax here.

File: assets/javascripts/discourse/templates/admin/plugins-explorer.hbs [hunk 2]
```diff
@@ -164,7 +164,7 @@
           </thead>
           <tr></tr>
           <tbody>
-            {{#each sortedQueries as |query|}}
+            {{#each searchedContent as |query|}}
```
Comment (techAPJ) [inline assets/javascripts/discourse/templates/admin/plugins-explorer.hbs]: Nitpicking.. but I think `filteredContent` looks better here?

**Comment by techAPJ** [comment]
Added a comment to use new syntax, other than that looks good. 👍

File: assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6 (hunk 1 above)
Comment (rishabhnambiar) [reply assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6]: Done, sorry!

File: assets/javascripts/discourse/templates/admin/plugins-explorer.hbs (hunk 2 above)
Comment (rishabhnambiar) [reply assets/javascripts/discourse/templates/admin/plugins-explorer.hbs]: Definitely, updated :+1:

**Pull request merged**